azurelinuxagent/common/future.py

```diff
@@ -25,16 +25,38 @@
 
     bytebuffer = memoryview # pylint: disable=C0103
 
+    # We aren't using these imports in this file, but we want them to be available
+    # to import from this module in others.
+    # Additionally, python2 doesn't have this, so we need to disable import-error
+    # as well.
+    from builtins import int, range # pylint: disable=unused-import,import-error
```

Comment:
Wont this throw if py2 doesnt have these?

Reply:
It's just off screen in the GitHub view, but we're in a `if sys.version_info[0] == 3:`, so this shouldn't be executed in a python2 env.

Reply:
Ahh got it! Thanks!